tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -1798,7 +1798,10 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 			pvc, err := client.CoreV1().PersistentVolumeClaims(namespace).Get(ctx, pvcName, metav1.GetOptions{})
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 			pv := getPvFromClaim(client, namespace, pvcName)
-			verifyBidirectionalReferenceOfPVandPVC(ctx, client, pvc, pv, fcdID)
+			pvName := pvc.Spec.VolumeName
+			//PVC volumename will be static-pv-<volumeID> This volumeID Should be same as in PV volumeHandle
```

Comment:
`pvc.Spec.VolumeName` refers to PV name AFAIK. Can you fix the comment

Reply:
updated